CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ We use labels to track, manage and organize issues and pull requests.
 | `needs review`     | [search][needs review]  | Work still needs to be reviewed by another member of the team. |
 | `#pairwithme`      | [search][#pairwithme]   | Looking for someone to pair on this. |
 | `question`         | [search][question]      | Feature and code related questions which are no bug reports or feature requests. |
-| `rgsoc-<year>`     | [search][rgsoc-<year>]  | Issues targeting a specific `<year>` of the program, being relevant to e.g. the beginning of the program in 2016. |
+| `rgsoc-<year>`     | [search][rgsoc-<year>]  | Issues that a pair of RGSoC students of a given `<year>` working on the Teams App as their project can work on. |
```

Comment:
Oh sorry, totally got that wrong @carpodaster 🙈 

As you say you don't like your new wording, how about: "Issues reserved for students working on the Teams App during RGSoC" - just to make it shorter?